.eslintrc.js

```diff
@@ -128,14 +121,19 @@ module.exports = {
     "implicit-arrow-linebreak": 0,
     // XXX Switch back to indent once mozilla-central has decided what it is using.
     "indent": 0,
-    "indent-legacy": ["error", 2, {"SwitchCase": 1}],
     "init-declarations": 0,
-    "jsx-quotes": [2, "prefer-double"],
     "line-comment-position": 0,
     "lines-around-comment": ["error", {
-      "allowClassStart": true,
+      "beforeBlockComment": true,
+      "afterBlockComment": true,
+      "beforeLineComment": true,
+      "afterLineComment": true,
+      "allowBlockStart": true,
+      "allowBlockEnd": true,
       "allowObjectStart": true,
-      "beforeBlockComment": true
+      "allowObjectEnd": true,
+      "allowArrayStart": true,
+      "allowArrayEnd": true
```

Comment:
Oh, it's probably not prettier adding the extra lines around comments. What was this change for?

Reply:
I reverted this "lines-around-comment" change from before the `--fix` commit and I don't see any conflicts with prettier for this rule ?

Reply:
https://github.com/prettier/eslint-config-prettier#lines-around-comment was the motivator here.  The lines-around-comment stuff still isn't working entirely correctly, and still needs debugging.

Reply:
The note there says to avoid conflicting configuration if `lines-around-comment` requires a blank line inside a block (before a comment). Our configuration doesn't do that for `// line comment`s but does for `/* block comment */` but our commenting style seems to only use block comments at top level or within classes (e.g., for methods), so our usage shouldn't require changes to this rule.

We could just add `"allowBlockStart": true` to be safe (and leave all the other `lines-around-comment` options the same) if someone did try to add a block comment at the beginning of a code block.

Reply:
In order to make things work, I ended up with this:

```
    "lines-around-comment": ["error", {
      "allowClassStart": true,
      "beforeBlockComment": false,
      "allowBlockStart": true,
    }],
```